docs/overview.md

```diff
@@ -0,0 +1,35 @@
+# Overview
```

Comment:
This seems to duplicate the README.  Why do we need both?  Is this linked from somewhere else?

Reply:
was wrestling with this myself, it's out of scope for the quickstart so i'll ditch it.